### PR TITLE
Pymotorcad_10_08

### PR DESCRIPTION
**Pull request by pvinod13**
Hi all, I have created Cheat sheet for PymotorCAD. Can you review it and merge if there are no conflicts.

### CODE REVIEW

**Comment by Revathyvenugopal162** [comment]
Thank you for the PR @pvinod13 , Pinging here @EstherProsperi  for proof reading. Can you please upload the the pdf of the rendered cheatsheet @pvinod13  and please review @EstherProsperi when ever you have some time.

**Comment by pvinod13** [comment]
[pymotorcad_cheat_sheet_updated.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12357147/pymotorcad_cheat_sheet_updated.pdf). Hi @EstherProsperi, please review the cheatsheet and give your feedback.

File: cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+data = io.loadmat(os.path.join(working_folder, mcad_name, "Lab", "MotorLAB_elecdata.mat"))
+speed = data["Speed"]
+shaft_torque = data["Shaft_Torque"]
+shaft_power = data["Shaft_Power"]
+\end{lstlisting}
+
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} PyMotorCAD Scripting in MATLAB}
+The Python version that MATLAB is using has the \textit{ansys.motorcad.core} package installed, PyMotorCAD is available to use in MATLAB.
+
+To import the ansys.motorcad.core module as pymotorcad for use in scripts
```
Comment (EstherProsperi) [inline cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex:160]: To import the \textit {ansys.motorcad.core} python package for use in MATLAB:

File: cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+\begin{lstlisting}[language=Python]
+pymotorcad = py.importlib.import_module('ansys.motorcad.core')	
+\end{lstlisting}
+
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Backwards Compatibility}
+Old ActiveX scripts use this code to connect to Motor-CAD
```
Comment (EstherProsperi) [inline cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex:166]: To use ActiveX scripts to connect to Motor-CAD:

File: cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+\begin{lstlisting}[language=Python]
+import win32com.client
+mcad = win32com.client.Dispatch("MotorCAD.AppAutomation")
+\end{lstlisting}
+To use PyMotorCAD to connect to Motor-CAD, you replace the preceding code with this code:
```
Comment (EstherProsperi) [inline cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex:171]: To use PyMotorCAD to connect to Motor-CAD:

**Comment by Revathyvenugopal162** [comment]
pinging here @PipKat based on her recent review.

**Review by Revathyvenugopal162 — APPROVED** [review]

File: cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+%\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Loading Template}
+%To load template and save file:
+%\begin{lstlisting}[language=Python]
+%mcad.load_template(template_name)
+%mcad.save_to_file(os.path.join(working_folder, mcad_name + ".mot"))
+%\end{lstlisting}
+
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Geometrical properties}
+To set the motor geometry:
```
[the remark this answers is not in the document]
Comment (PipKat) [reply cheat_sheets/pymotorcad_cheat_sheet/pymotorcad_cheat_sheet.tex]: @Revathyvenugopal162 Why is the "Commit suggestion" button unavailable? Based on you pinging me here, it looks like you'd like me to review this after Esther to make the same types of changes I did in [PR #62](https://github.com/ansys/pyansys-cheat-sheet/pull/62/). Please confirm. Thanks!

**Comment by PipKat** [comment]
@Revathyvenugopal162 @pvinod13 @EstherProsperi I am going to merge this PR and create a new one in which I will edit the cheat sheet. Thanks, all.

**Pull request merged**